crt_portal/cts_forms/templates/forms/index.html

```diff
@@ -25,65 +25,64 @@ <h1>
     <a class="usa-button margin-top-2" href="/report" label="first step" name="Create record">
         + Add new record
     </a>
-
     <p>{{ data_dict|length }} of {{ page_format.count }} records</p>
-
     {% include "forms/snippets/pagination.html" with page_format=page_format %}
-
   </div>
-    <table class="usa-table crt-table">
-      <thead>
-        <tr>
-          <th scope="col">Status</th>
-          <th scope="col">Routed</th>
-          <th scope="col">Submitted</th>
-          <th scope="col">Contact Name</th>
-          <th scope="col">Contact Details</th>
-          <th scope="col">Summary</th>
-          <th scope="col">Class</th>
+  <table class="usa-table crt-table">
```

Comment:
Looks like a lot of changes, but this is just aligning HTML indentation with the previous element.